GitUI/CommandsDialogs/FormDiff.cs

```diff
@@ -65,6 +71,12 @@ public partial class FormDiff : GitModuleForm
             DiffText.ExtraDiffArgumentsChanged += DiffTextOnExtraDiffArgumentsChanged;
         }
 
+        protected override void OnRuntimeLoad(EventArgs e)
+        {
+            _revisionDiffContextMenuController = new FileStatusListContextMenuController();
```

Comment:
:question:  Can this be moved into constructor? If so, this method an go....

Reply:
Yes, I just followed the pattern you had use in other modules.

Reply:
I would have used `OnRuntimeLoad` instead of instantiating objects in constructor where would be a dependency on fully set `GitModule`, which isn't available until the control is loaded.
In this case there is no dependency on `GitModule` and I'm pretty sure it is safe to move the instantiation in to the constructor.

Reply:
Will move